Backend/app/rider-platform/rider-app/Main/src/Storage/Beam/DisabilityTranslation.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-missing-signatures #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Storage.Beam.DisabilityTranslation where
+
+import Data.Serialize
+import qualified Database.Beam as B
+import Database.Beam.MySQL ()
+import EulerHS.KVConnector.Types (KVConnector (..), MeshMeta (..), primaryKey, secondaryKeys, tableName)
+import GHC.Generics (Generic)
+import Kernel.Beam.Lib.UtilsTH
+import Kernel.Prelude hiding (Generic)
+import Sequelize
+
+data DisabilityTranslationT f = DisabilityTranslationT
+  { id :: B.C f Text,
```

Comment:
{disabilityId, tag, description, language} where (disabilityId and language) will be the primary keys.

Reply:
Removed id from `disability_translation` table and made composite key (disabilityId and language) as the primary key for `disability_translation` table.